moveit_ros/perception/semantic_world/src/semantic_world.cpp

```diff
@@ -146,7 +146,7 @@ bool SemanticWorld::addTablesToCollisionWorld()
       triangles[i3] = j + 1;
     }
 
-    shapes::Shape* table_shape = shapes::createMeshFromVertices(vertices, triangles);
+    shapes::Shape* table_shape = shapes::createMeshFromVertices(vertices, triangles); // We could move this part to a new function to keep the function pure
```

Comment:
I rather would see you simply open a PR to make this change :+1:

```suggestion
    shapes::Shape* table_shape = shapes::createMeshFromVertices(vertices, triangles);
```

Reply:
OK, sir, I'll work on it this week inshallah.

Reply:
Thank you for your time. This is not possible for me at the moment actually as I don't have a whole idea about the code, I'll only work in small pieces with the help of my simple knowledge of functional programming. 
Soon this will be possible inshaAllah.